### DISCUSSION TITLE
Welcome to range-v3 Discussions!

### DISCUSSION THREAD

**Discussion by ericniebler** [ericniebler/range-v3#1600] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by JohelEGP** [comment]
Greetings. I'm a hobbyist C++ programmer. I hope to help build abstractions that improve readability. And that they become zero cost pending the optimization from compilers. And looking forward to the coming of the pizza `|>` operator.

**Comment by zeroxia** [comment]
Is it possible to support C++11?
Thanks.
